src/styles/About.styles.js

```diff
@@ -63,7 +63,7 @@ export const GuidingPrinciple = styled.div`
 
 export const HeroHeadline = styled.span`
   ${({ black }) =>
-    black ? `color: ${colors.black};` : `color: ${colors.white}`}
+    black ? `color: ${colors.black};` : `color: ${colors.black}`}
```

Comment:
if this logic isn't necessary anymore you can just remove it + render the text as normal/without spans!